src/index.js

```diff
@@ -95,11 +95,11 @@ interface Platform {
   /**
   * The runtime's location API.
   */
-  location: Object;
+  location: typeof window.location;
   /**
   * The runtime's history API.
   */
-  history: Object;
+  history: typeof window.history;
```

Comment:
@EisenbergEffect Yeah, this is correct. `typeof` in TypeScript doesn't mean "the class of X", but simply "same type as X". So "typeof window.history" is the type of an instance of history.